docs/book/src/topics/custom-images.md

```diff
@@ -104,7 +104,8 @@ Managed images support only 20 simultaneous deployments, so for most use cases S
 
 ### Using Azure Marketplace
 
-To use an image from [Azure Marketplace][azure-marketplace], populate the `publisher`, `offer`, `sku`, and `version` fields:
+To use an image from [Azure Marketplace][azure-marketplace], populate the `publisher`, `offer`, `sku`, and `version` fields, and if this image is published by a third 
+party publisher, set the `thirdPartyImage` flag to `true` so an image plan can be generated for it. 
```

Comment:
let's also document the need to accept offer terms on the subscription before being able to use it if the image is third party

Reply:
updated